app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topic-controller");
+
+const app = express();
+
+app.use(express.json());

```

Comment:
At this point, this line isn't needed as you aren't posting anything - no doubt you'll need it in the future, but not just yet!